e2e_tests/integrations/test_lightning.py

```diff
@@ -33,6 +34,15 @@
 import neptune.new as neptune
 from e2e_tests.base import BaseE2ETest
 
+LIGHTNING_ECOSYSTEM_ENV_PROJECT = "NEPTUNE_LIGHTNING_ECOSYSTEM_CI_PROJECT"
+
+skip_if_on_regular_env = pytest.mark.skipif(
+    LIGHTNING_ECOSYSTEM_ENV_PROJECT not in os.environ, reason="Tests weren't invoked in Lightning Ecosystem CI"
+)
+skip_if_on_lightning_ecosystem = pytest.mark.skipif(
+    LIGHTNING_ECOSYSTEM_ENV_PROJECT in os.environ, reason="Tests invoked in Lightning Ecosystem CI"
```

Comment:
This will be true even if we set `NEPTUNE_LIGHTNING_ECOSYSTEM_CI_PROJECT=0` it's better to rely on  boolean value with default fallback instead of existence of the key.
@Raalsky we had similar problem recently with non detecting `False` value of envs and have some helper methods for that.